### PR TITLE
Fix docker push

### PR DESCRIPTION
**Pull request by jgadling**
the ECR-push github action was broken - this fixes it. I tested it on a branch a few minutes ago.

### CODE REVIEW

File: .github/workflows/push-ingestor-build.yaml [hunk 1]
```diff
@@ -27,10 +27,16 @@ jobs:
...
-        env:
-          ECR_REPO: ${{ secrets.ECR_REPO }}
-          AWS_REGION: ${{ secrets.AWS_REGION }}
-        run: make push-ingestor-build ecr_repo=$ECR_REPO tag=latest aws_region=$AWS_REGION
+      - name: Docker build & Push
+        uses: docker/build-push-action@v4
+        with:
+          context: ./ingestion_tools
+          file: ./ingestion_tools/Dockerfile
+          tags: ${{ secrets.ECR_REPO }}:${{ github.head_ref || github.ref_name }}
```
Comment (manasaV3) [inline .github/workflows/push-ingestor-build.yaml:40]: As this is triggered on push to main, `github.head_ref` might always turn up empty.

**Review by manasaV3 — APPROVED** [review]
Should we also set the tag in the enqueue script to default to `main`?

**Comment by jgadling** [comment]
> Should we also set the tag in the enqueue script to default to `main`?

Good idea! I'll update it!

File: .github/workflows/push-ingestor-build.yaml (hunk 1 above)
Comment (jgadling) [reply .github/workflows/push-ingestor-build.yaml:40]: I tested it on another branch, and the branch name came through, so 🤞 I'm expecting this to resolve to `main` in most cases

**Pull request merged**